### ISSUE TITLE
about the defensegan pre-training model

### ISSUE THREAD

**Issue by ZackSoul** [mnswdhw/InvGAN-Pytorch#3]
Do you have the defensegan's pre-training model of pytorch? I'm trying to reproduce it, but the accuracy of my gan network reconstruction after training is very low. If you have the pre-training model, it will help me a lot. Thank you very much.

**Comment by mnswdhw** [comment]
I have given some conversion scripts. Try to follow them to easily obtain Pytorch compatible model weights. I have already implemented the corresponding architecture that can load those weights. I do not have the weights currently, but I think they can be obtained easily using the conversion scripts. Let me know if you have any problem following them!
